<launch>
    <!-- The frame of the sensor used to obtain the data to be filtered; This
         parameter is optional. If it is not specified, shadow
         points will be considered outside -->
    <param name="sensor_frame" value="" />

    <!-- Minimum distance to sensor (for point not to be considered inside) -->
    <param name="min_sensor_dist" value="0.01" />

    <!-- The padding to be added for the body parts the robot can see -->
    <param name="self_see_padd" value="0.02" />

    <!-- The scaling to be added for the body parts the robot can see -->
    <param name="self_see_scale" value="1.0" />

    <!-- The names of the links the sensor can see -->
    <rosparam>
      self_see_links:
      - {name: 'arm_1_link', padding: 0.1, scale: 1.0}
      - {name: 'arm_2_link', padding: 0.1, scale: 1.0}
      - {name: 'arm_3_link', padding: 0.1, scale: 1.0}
    </rosparam>
  </launch>
